### PR TITLE
[Buster] Apply example_ynh

### LINKED ISSUE #40: [Buster] Install fail miserably because of usage of ynh_systemd_action
Didn't dig very far, maybe due to the custom definition in _common.sh ?

https://ci-buster.nohost.me/ci/apps/lufi/

```
ynh_systemd_action -n $app -a start -l "Server available at" -p "systemd"
[...]
Warning: yunohost.hook <lambda> - [1020.1] [WARN] Too many arguments ! "-p" will be ignored.
Warning: yunohost.hook <lambda> - [1020.1] [WARN] Too many arguments ! "systemd" will be ignored.
Warning: yunohost.hook <lambda> - [1020.1] Unknown operation lstu.
Warning: yunohost.hook <lambda> - [1020.1] Failed to parse lines 'Server'
```

### PR DESCRIPTION
**Pull request by yalh76**
- *Fix #40*

- *Apply example_ynh*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [x] Fix or enhancement tested.
- [x] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Package_check results
---
[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/lstu_ynh%20PR41%20(yalh76)/)  

Own CI results for Revision: e318727e47d89f9bdc25f51fc7d2cb777aa7bcca:

## Stretch
running

## Buster
running

### CODE REVIEW

File: scripts/change_url [hunk 1]
```diff
@@ -89,48 +105,55 @@ fi
...
+ynh_store_file_checksum --file="$config"
 
 #=================================================
 # GENERIC FINALISATION
+#=================================================
+# START SYSTEMD SERVICE
+#=================================================
+ynh_print_info --message="Starting a systemd service..."
+
+ynh_systemd_action --service_name=$app --action="start" --log_path="systemd" --line_match="Starting hot deployment for Hypnotoad server"
```
Comment (maniackcrudelis) [inline scripts/change_url]: Tested on Buster, everything work except that line is never found.
Would probably be easier to look for "Server available at"

Comment (yalh76) [reply scripts/change_url]: Changed, waiting for package_check results

Comment (yalh76) [reply scripts/change_url]: Change made

File: scripts/actions/change_password [hunk 2]
```diff
@@ -29,7 +29,7 @@ hashed_password_old=$(ynh_app_setting_get $app hashed_password)
 
 if [ "$hashed_password" == "$hashed_password_old" ]
 then
-	ynh_die "Same password." 0
+	ynh_die --message="Same password." 0
```
Comment (kay0u) [inline scripts/actions/change_password]:
```suggestion
	ynh_die --message="Same password." --ret_code=0
```

File: scripts/actions/change_theme [hunk 3]
```diff
@@ -28,7 +28,7 @@ theme_old=$(ynh_app_setting_get $app theme)
 
 if [ "$theme" == "$theme_old" ]
 then
-	ynh_die "Same theme." 0
+	ynh_die --message="Same theme." 0
```
Comment (kay0u) [inline scripts/actions/change_theme]:
```suggestion
	ynh_die --message="Same theme." --ret_code=0
```

File: scripts/actions/public_private [hunk 4]
```diff
@@ -41,7 +41,7 @@ is_public_old=$(ynh_app_setting_get $app is_public)
 
 if [ $is_public -eq $is_public_old ]
 then
-	ynh_die "is_public is already set as $is_public." 0
+	ynh_die --message="is_public is already set as $is_public." 0
```
Comment (kay0u) [inline scripts/actions/public_private]:
```suggestion
	ynh_die --message="is_public is already set as $is_public." --ret_code=0
```

**Review by kay0u — COMMENTED** [review]
LGTM otherwise

**Review by alexAubin — APPROVED** [review]
Quick review, LGTM :+1:

**Pull request merged**